### DISCUSSION TITLE
Downloading protein ids (or gene ids) and their corresponding molecular function.

### DISCUSSION THREAD

**Discussion by igregore-cst** [geneontology/helpdesk#374] [Q&A]
Could a kind soul point me in the right direction?

I am trying to download a table with two columns:
"protein id" (or gene id) and
"molecular function"

For all proteins known to geneontology.org.

That's it.

Is there a portal for big file downloads for me to get and subset or filter?

Should I use a web portal perform the proper query and export the table on demand?

I did try to answer my own questions but I failed.

Thank you.

Ivan

**Comment by suzialeksander** [comment]
Hi @igregore-cst, 

We have some resources listed at http://geneontology.org/docs/download-go-annotations/, including the API at http://api.geneontology.org/api. 

AmiGO does allow downloads and custom downloads, but has a limit of 100000- if you want to go this route you may need to use additional filters and download several files to get the complete table.
